packages/domains-table/src/domains-table/style.scss

```diff
@@ -137,12 +142,48 @@
 		}
 	}
 
+	.domains-table-row__ssl-status-button {
+		text-decoration: none;
+
+		&:focus {
+			box-shadow: none;
+		}
+
+		&.is-active,
+		&.is-active :hover {
+			color: var(--studio-green-50);
+		}
+		&__pending,
+		&__pending :hover,
+		&__disabled,
+		&__disabled :hover {
+			color: var(--studio-orange-50);
+		}
```

Comment:
I see you already caught this in fea46167dd870d9e6ad9176f258ff6c208c1c179, @chad1008 👍

Reply:
Thanks for spotting this, I hadn't meant to leave two different naming conventions in place. Initially I transitioned from BEM to `is-<status>` because BEM specificity was struggling to overcome the Button component's base styles. I also liked the brevity of it as well.

That said, I think your point about consistency is a good one. I've updated the styles to use BEM exclusively, and gotten them specific enough that we don't have any unexpected hover behavior. It's a bit more verbose, but I think consistency within the file is the way to go.